app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -406,85 +111,11 @@
-              {{/if}}
-            {{/if}}
-            {{#if (eq field.type 'image')}}
-              <Widgets::Forms::ImageUpload
-                @imageUrl={{mut (get this.data.session field.fieldIdentifier)}}
-                @label={{t "Logo"}}
-                @id={{if field.isRequired (concat "session_" field.fieldIdentifier "_required") (concat "session_" field.fieldIdentifier)}}
-                @icon="image"
-                @hint={{t "Select an image"}}
-                @maxSizeInKb={{1000}} />
-            {{/if}}
-            {{#if (eq field.type 'file')}}
-              <Widgets::Forms::FileUpload
-                @fileUrl={{mut (get this.data.session field.fieldIdentifier)}}
-                @label={{t "File"}}
-                @id={{if field.isRequired (concat "session_" field.fieldIdentifier "_required") (concat "session_" field.fieldIdentifier)}}
-                @icon="file"
-                @hint={{t "Select a file"}}
-                @maxSizeInKb={{10000}} />
-            {{/if}}
-            {{#if (eq field.type 'select')}}
-              {{#if (eq field.fieldIdentifier 'language')}}
-                <UiDropdown
-                  @class="search selection"
-                  @selected={{this.data.session.language}}
-                  @forceSelection={{false}}
-                  @fullTextSearch={{true}}>
-                  <Input
-                    @type="hidden"
-                    @id="session_language"
-                    @value={{this.data.session.language}} />
-                  <i class="dropdown icon"></i>
-                  <div class="default text">{{t 'Select'}}</div>
-                  <div class="menu">
-                    {{#each this.languages as |language|}}
-                      <div class="item" data-value="{{language.name}}">
-                        {{language.name}}
-                      </div>
-                    {{/each}}
-                  </div>
-                </UiDropdown>
-              {{else}}
-                <UiDropdown
-                  @class="search selection"
-                  @onChange={{action (mut (get this.data.session field.fieldIdentifier))}} as |execute mapper|>
-                  <i class="dropdown icon"></i>
-                  <div class="default text">
-                    {{if (get this.data.session field.fieldIdentifier) (get this.data.session field.fieldIdentifier) (concat 'Select a ' field.name)}}
-                  </div>
-                  <div class="menu">
-                    {{#each (get this.data (concat field.fieldIdentifier 's')) as |item|}}
-                      <div class="item" data-value="{{map-value mapper item}}">{{item.name}}</div>
-                    {{/each}}
-                  </div>
-                </UiDropdown>
-              {{/if}}
-            {{/if}}
-          </div>
-        {{/if}}
-      {{/each}}
+      <Forms::SessionSection
+        @fields={{this.allFields.session}}
+        @data={{this.data}} />
     {{/if}}
   {{/if}}
```

Comment:
just add a line break here. or mt-4 in button class. just to give some space above submit button and forms.